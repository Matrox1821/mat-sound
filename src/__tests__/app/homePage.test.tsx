import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Page from "@/app/page";
import { writeHeapSnapshot } from "v8";

describe("Page", () => {
  it("renders a heading", () => {
    render(<Page />);

    expect(screen.getByText(/hola/i)).toBeInTheDocument();
  });
  it("snapshot", () => {
    const { asFragment } = render(<Page />);
    const firstRender = asFragment();
    expect(firstRender).toMatchSnapshot();
  });
});
