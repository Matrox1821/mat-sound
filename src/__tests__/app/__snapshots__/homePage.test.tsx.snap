// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col items-center justify-items-center min-h-screen w-full p-8 bg-accent/10 "
  >
    <a
      href="/admin"
    >
      Admin
    </a>
    <a
      href="/admin/create/tracks"
    >
      Admin-Create-Track
    </a>
    <span
      class="border"
    >
      Hola farbits
    </span>
  </div>
</DocumentFragment>
`;
